github/pulls_comments.go

```diff
@@ -79,6 +81,7 @@ func (s *PullRequestsService) ListComments(ctx context.Context, owner string, re
 
 	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeReactionsPreview)
+	req.Header.Add("Accept", mediaTypeMultiLineCommentsPreview)
```

Comment:
This is not how we set the Accept header when multiple fields are needed. Here is an example:

```
	// TODO: remove custom Accept header when this API fully launches.
	acceptHeaders := []string{mediaTypeLabelDescriptionSearchPreview, mediaTypeLockReasonPreview, mediaTypeDraftPreview}
	req.Header.Set("Accept", strings.Join(acceptHeaders, ", "))
```